src/components/common/Sidebar/Sidebar.js

```diff
@@ -38,8 +39,21 @@ const Sidebar = ({ children, userProfile }) => {
 
   // This is determining which role is currently in session, implemented further in ternary statements in the return clause
   // const isUserMentee = useMemo(() => role_id === 4, [role_id]);
-  const isUserMentor = useMemo(() => role_id === 3, [role_id]);
+
+  /**
+   * Khaleel Musleh
+   * Changed role_id to strict equality and removed the ELSE statement of mentee as it was crashing whenever we update the role_id or add a new role_id such as Dev role ID
+   *  while creating an IF statement for each role
+   */
+
   const isUserAdmin = useMemo(() => role_id <= 2, [role_id]);
+  const isUserMentor = useMemo(() => role_id === 3, [role_id]);
+  const isUserMentee = useMemo(() => role_id === 4, [role_id]);
+  /**
+   * Khaleel Musleh
+   * Created a Dev role_id which displays all the sidebar links of Admin, Mentor, Mentee
+   */
+  const isUserDev = useMemo(() => role_id === 5, [role_id]);
```

Comment:
is role_id===1 reserved only for superAdmin? Curious about 2-5 approach.

Reply:
Hi Ryan, Ive made another strict equality check for super admin which is role_id === 1
they all now have a check from 1 - 5